packages/react-charting/src/utilities/ChartHoverCard/ChartHoverCard.styles.ts

```diff
@@ -56,7 +56,7 @@ export const getChartHoverCardStyles = (props: IChartHoverCardStyleProps): IChar
     calloutContentY: [
       theme.fonts.xxLarge,
       {
-        color: color ? color : theme.semanticColors.bodyText,
+        color: theme.semanticColors.bodyText,
```

Comment:
This change needs to happen only to the placeholder callout y text, not for all scenarios.

Reply:
Made this change because multi value callout in cartesian chart displays y value in the default body text color.
If this change is only needed for placeholder bar, we will need to add/use another prop, as the color prop is also used by the vertical line. Should I proceed with this change?

Reply:
You can just set the color property to bodyText in case of placeholder bar. Need not add a new property.

Reply:
done